src/components/FeaturesDemo/WebBrowser/ReportPage/AssertViewResult.tsx

```diff
@@ -0,0 +1,103 @@
+import clsx from "clsx";
+import React from "react";
+
+import { CollapsibleInfo } from "@site/src/components/FeaturesDemo/WebBrowser/ReportPage/CollapsibleInfo";
+import {
+    AssertViewResultWithState,
+    Status,
+} from "@site/src/components/FeaturesDemo/WebBrowser/ReportPage/types";
+import { ActionButton } from "@site/src/components/FeaturesDemo/WebBrowser/ReportPage/ActionButton";
+import TickIcon from "@site/static/icons/tick.svg";
+import SearchIcon from "@site/static/icons/search.svg";
+import FullscreenIcon from "@site/static/icons/fullscreen.svg";
+import CompareIcon from "@site/static/icons/compare.svg";
+import UndoIcon from "@site/static/icons/undo.svg";
+import "./styles.module.scss";
+
+interface AssertViewResultProps {
+    data: AssertViewResultWithState;
+}
+
+export function AssertViewResult({ data }: AssertViewResultProps) {
+    // TODO: animated diff bubbles
+
+    return (
+        <CollapsibleInfo
+            title={data.stateName}
+            isCollapsed={data.isCollapsed}
+            titleClassName={clsx({
+                "text-green-600": data.status === Status.Success,
+                // "text-black": data.status === Status.NotLaunched,
+                "text-red-600": data.status === Status.Diff,
+            })}
+        >
+            <div className="my-1 flex flex-wrap">
+                {data.status === Status.Diff && (
+                    <>
+                        <ActionButton dataRoleId="accept-button" className="mr-1">
+                            <TickIcon className="mr-1 size-3" />
+                            Accept
+                        </ActionButton>
+                        <ActionButton className="mr-1">
+                            <SearchIcon className="size-3 sm:mr-1" />
+                            <span className="hidden sm:inline">Find same diffs</span>
+                        </ActionButton>
+                        <ActionButton className="mr-1">
+                            <FullscreenIcon className="size-3 sm:mr-1" />
+                            <span className="hidden sm:inline">Fast accept mode</span>
```

Comment:
why are you using the `hidden` value here?

Reply:
In order to free up as much space as possible on small phone screens. `hidden sm:inline` means that the block won't visible on mobile screens, but on larger ones will be inline.